public/locales/de/translation.json

```diff
@@ -19,10 +19,11 @@
     "about": "Über",
     "support": "Unterstützung",
     "english": "English",
-    "spanish": "Spanisch",
+    "spanish": "Español",
```

Comment:
We removed the Spanish language from the Portal. You can delete these translation properties. Sorry for leaving them in :)